mods/hv/rules/terrain.yaml

```
@@ -207,31 +238,48 @@ TreeAsh15:
 		Offset: 0,1
 		Template: 1476
 
-TreeMud15:
-	Inherits: ^TreeMud
-	LaysTerrain:
-		Template: 1475
-		Offset: 0,1
+TREEMUD:
+	Inherits: ^Rock
+	-Building:
+	-HiddenUnderShroud:
+	AlwaysVisible:
+	Immobile:
+		OccupiesSpace: false
+	-MapEditorData:
 
-TreeMudSnow16:
-	Inherits: ^TreeMud
-	LaysTerrain:
-		Offset: 0,1
-		Template: 1432
+TREEASH2:
```

```suggestion
TREEASH:
```

There is no `TREEASH` actor.